src/test/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteClientTest.java

```diff
@@ -107,4 +109,16 @@ public void testProjectConfiguration()
         Assert.assertFalse(uploadedProject.isEnabled());
         Assert.assertEquals(1, this.mockConnection.tasksForChallenge(TEST_CHALLENGE).size());
     }
+
+    @Test
+    public void testWriteChallengeIdsToFile()
+    {
+        final String fileName = String.format("%s/src/test/resources/challenges/challengeIds",
+                System.getProperty("user.dir"));
+        final Optional<String> fileNameOptional = Optional.of(fileName);
+        assertTrue(fileNameOptional.isPresent());
+        MapRouletteClient.setOutputPath(fileNameOptional);
+        MapRouletteClient.writeChallengeIdsToFile(1, 24);
+        MapRouletteClient.writeChallengeIdsToFile(1, 25);
```

Comment:
I think we should also verify that file file is created, and the contents in the file are what we expect

Reply:
For sure.